packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -516,3 +524,118 @@ export function useOnMount(onMount?: TLOnMountHandler) {
 		if (editor) return onMountEvent?.(editor)
 	}, [editor, onMountEvent])
 }
+
+const WATERMARK_SRC = `url('data:image/svg+xml;utf8,${encodeURIComponent(watermarkDesktopSvg)}') center 100% / 100% no-repeat`
+
+const Watermark = React.memo(() => {
+	const events = useCanvasEvents()
+
+	const ref = useRef<HTMLAnchorElement>(null)
+	useLayoutEffect(() => {
+		// eslint-disable-next-line deprecation/deprecation
+		if (ref?.current) ref.current.style.webkitMask = WATERMARK_SRC
+	}, [])
+
+	const editor = useEditor()
+	const showWatermark = useValue(
+		'show watermark',
+		() => {
+			const { width } = editor.getViewportScreenBounds()
+			if (width < 760) return false
```

Comment:
why 760? we should keep parity with:
`export const PORTRAIT_BREAKPOINTS = [0, 390, 428, 468, 580, 640, 840, 1023]`

Reply:
The breakpoints are at the UI layer. I picked a number semi-arbitrarily (that still fits with out UI).

Reply:
Ok, gotcha - I think that's worth a comment then... feels like this'll break at some point if/when we shift the UI and we'll forget about this.